tableau_utilities/__init__.py

```diff
@@ -1,5 +1,5 @@
-from .main import do_args
-from .main import main
+# from tableau_utilities.scripts.main import do_args
```

Comment:
@JustinGrilli  I don't understand what these 2 lines are for.  The new CLI code I have works without this.  Can we delete these?

Reply:
Hmm I may want to test a couple of things around this, but you are probably correct that these imports are not needed.